src/emulator/modules/SceDisplay/SceDisplay.cpp

```diff
@@ -115,15 +136,32 @@ EXPORT(int, sceDisplayWaitVblankStart) {
 }
 
 EXPORT(int, sceDisplayWaitVblankStartCB) {
-    return UNIMPLEMENTED();
+    display_wait(host);
+
+    if (host.display.abort.load())
+        return SCE_DISPLAY_ERROR_NO_PIXEL_DATA;
+
+    return SCE_DISPLAY_ERROR_OK;
 }
 
-EXPORT(int, sceDisplayWaitVblankStartMulti) {
-    return UNIMPLEMENTED();
+EXPORT(int, sceDisplayWaitVblankStartMulti, SceUInt32 uiVcount) {
+    if (uiVcount == 1)
```

Comment:
remove the `if`, better to let it wait 1 for every case

Reply:
my search not said that, but only return this function if Vcount is 1, and wait if get 2